src/shared/pipes.tsx

```diff
@@ -48,3 +48,16 @@ export const endDatePipe: Function = (startDate: Date, durWeeks: number): string
   endDate.setDate(endDate.getDate() + durWeeks * 7);
   return endDate.toLocaleDateString();
 };
+
+// pipes to add;
+// Empty string pipe
+export const emptyStringPipe: Function = (str: string): string => {
+  return (str === undefined || str === null) ? "" : str;
+};
+
+// EM dash pipe
+export const emDashPipe: Function = (str: string): string => {
+  return (str === undefined || str === null) ? "—" : "str";
```

Comment:
I think you meant the variable str and not the string str?

Reply:
Is the convention with pipes to create a variable inside that takes in some data, to then be used to do x task?

Reply:
```suggestion
  return (str === undefined || str === null) ? "—" : str;
```
Clarified with the above suggestion